src/Phinx/Console/Command/Migrate.php

```diff
@@ -104,6 +104,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln('<info>using table suffix</info> ' . $envOptions['table_suffix']);
         }
 
+        $versionOrder = $this->getConfig()->getVersionOrder();
+        $output->writeln('<info>ordering by </info>' . $versionOrder . " time");
```

Comment:
why /info closing before the rest of the line?

Reply:
This is a copy of what is going on in the rollback command which has had that since bba44b381fef9756c1d2f5a6c65ddc6757e7f6bb. I can update both to have a different format if so desired.

Reply:
I think I would have expected more like 

    <info>ordering by</info> ...

The whitespace after the closing tag.
But no big deal.

Reply:
Fixed in both commands.